app/src/main/java/org/jellyfin/androidtv/playback/overlay/CustomSeekProvider.java

```diff
@@ -0,0 +1,27 @@
+package org.jellyfin.androidtv.playback.overlay;
+
+import androidx.leanback.widget.PlaybackSeekDataProvider;
+
+public class CustomSeekProvider extends PlaybackSeekDataProvider {
```

Comment:
I'm not really sure what this class does. Is it providing the possible "seek positions" to the player? And if so, does this mean that whatever I watch it can only seek to 30 positions? Which would mean that if I watched a movie of 30 hours I would skip one hour every time?

Reply:
I just confirmed this myself while testing. I think adding stepts for every `x` seconds would be better.

Reply:
Yes, I also think I saw a code example of a quick seeker, which means the faster you click (or the longer you hold to the right), the faster you seek

Reply:
Can you update this one so it works better for movies / long episodes? 30 steps is enough for a 30 minute show but it isn't for a 3 hour movie. The behavior you mentioned you saw in an example would be awesome to have